containers/Dockerfile.EESSI-client-pilot-centos7-ppc64le

```diff
@@ -0,0 +1,27 @@
+FROM docker.io/ppc64le/centos:7
+
+RUN yum install -y sudo vim openssh-clients cmake wget make unzip patch valgrind bzip2
+
+RUN yum install -y gcc gcc-c++ sqlite-devel python-devel libcap-devel libuuid-devel attr \
+ && yum install -y fuse-devel fuse3-devel zlib-devel openssl-devel
+
+# build CernVM-FS from source (no aarch64 Debian packages available)
+RUN  wget https://github.com/cvmfs/cvmfs/archive/cvmfs-2.7.5.tar.gz && \
+  tar xfz cvmfs-2.7.5.tar.gz && \
+  cd cvmfs*2.7.5/ && \
+  mkdir build && \
+  cd build && \
+  cmake .. -DBUILD_SERVER=no -DBUILD_SERVER_DEBUG=no -DDBUILD_SHRINKWRAP=no && \
+  make -j $(nproc) && \
+  sudo make install && \
+  cd / && \
+  rm -r cvmfs*2.7.5*
+
+RUN yum install -y https://github.com/EESSI/filesystem-layer/releases/download/v0.2.3/cvmfs-config-eessi-0.2.3-1.noarch.rpm
```

Comment:
Looking at variables, should we have a variable for the config version and possibly the config package version (as they don't always match due to `-1`)?

Reply:
That makes sense too. We also were thinking about having some link that always points to the latest version, which is partly already made possible by Github, except for the version in the filename...